internal/lsp/server.go

```diff
@@ -884,9 +883,7 @@ func (l *LanguageServer) StartCommandWorker(ctx context.Context) { //nolint:main
 							value = make(map[string]any)
 						}
 
-						json := encoding.JSON()
-
-						jsonVal, err = json.MarshalIndent(value, "", "  ")
+						jsonVal, err = encoding.JSON().MarshalIndent(value, "", "  ")
```

Comment:
this is unrelated and just a refactor, I think it's correct?

Reply:
I think that might be remains of the first implementation, and me trying to make everything look more or less the same as before. That should of course have been an import alias if so thugh. Now I consider it more of a feature that it looks like different from stdlib `json`. Well, at least until the `v2` package arrives 🙂